.github/workflows/master-e2e.yaml

```
@@ -110,6 +110,9 @@ on:
       turtles_operator_version:
         description: Turtles Operator Version; only to be used when dev chart is set to false; default - latest stable version
         type: string
+      skip_deletion_test:
+        description: Skip deletion tests to retain the clusters
```

```suggestion
      skip_cluster_deletion:
        description: Skip CAPI cluster and fleet repo deletion
```
IMHO, this nomenclature seems appropriate to denote retention of clusters